index.php

```diff
@@ -8,8 +8,8 @@
 
 ?>
 
-<!DOCTYPE HTML>
-<HTML lang="en">
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
nice but you missed out the closing tag haha

Reply:
done now!